x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/cytoscape_options.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import cytoscape from 'cytoscape';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import {
+  ANALYSIS_CONFIG_TYPE,
+  JOB_MAP_NODE_TYPES,
+} from '../../../../../../common/constants/data_frame_analytics';
+import classificationJobIcon from './icons/ml_classification_job.svg';
+import outlierDetectionJobIcon from './icons/ml_outlier_detection_job.svg';
+import regressionJobIcon from './icons/ml_regression_job.svg';
+
+const lineColor = '#C5CCD7';
+// @ts-expect-error `documentMode` is not recognized as a valid property of `document`.
+const isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
```

Comment:
I recall we don't support IE11 anymore. And to avoid using `@ts-expect-error` try `document['documentMode']` or extend the interface
```typescript
declare global {
    interface Document {
        documentMode?: any;
    }
}
```

Reply:
Removed in 1bafe2e088ed25a67b279bc087aec2c3ecbc02d1